<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestsProgr2022</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:136.4, stop:0.505682 rgba(252, 129, 129, 255), stop:0.761364 rgba(118, 129, 129, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>49</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Palatino Linotype</family>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TestsProgr</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="but_reg">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>240</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(252, 129, 129);</string>
   </property>
   <property name="text">
    <string>Зарегестрироваться</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_sign">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>270</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(252, 129, 129);</string>
   </property>
   <property name="text">
    <string>Войти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_create_test">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>190</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(252, 129, 129);</string>
   </property>
   <property name="text">
    <string>Создать тест</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_info">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>369</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(211, 108, 108);</string>
   </property>
   <property name="text">
    <string>Инфо</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_text">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>261</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 160, 160, 150);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="but_info_back">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(252, 129, 129);</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>408</x>
     <y>379</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>by RubyRed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Palatino Linotype</family>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TestsProgr</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_reg_login">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>147</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_reg_password_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>228</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_reg_password">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>187</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="text_reg_login">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Логин</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text_reg_password">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>171</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text_reg_password_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>211</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Пароль повтор</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="but_reg_enter">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>262</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(252, 129, 129);</string>
   </property>
   <property name="text">
    <string>Готово</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_reg_back">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>292</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(252, 129, 129);</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_sign_password">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>182</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_sign_password">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>198</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="text_sign_login">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>142</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Логин</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_sign_login">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>158</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="but_sign_enter">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>229</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(252, 129, 129);</string>
   </property>
   <property name="text">
    <string>Готово</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_new_test">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>182</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(252, 129, 129);</string>
   </property>
   <property name="text">
    <string>Новый</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_old_test">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(252, 129, 129);</string>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_sign_back">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>259</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(252, 129, 129);</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_tmain_back">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(252, 129, 129);</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_name_test">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_name_test">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Название теста(минимум 3 символа)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_next">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(199, 129, 129);</string>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_pre">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(199, 129, 129);</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_text_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>261</width>
     <height>171</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Точно выйти?&lt;br/&gt;Не сохранёные даные будут&lt;/p&gt;&lt;p&gt;безвозвратно утерены!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 160, 160, 250);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Точно выйти?
Не сохранёные даные будут
безвозвратно утерены!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="but_exit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 160, 160, 250);</string>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_no_exit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 160, 160, 250);</string>
   </property>
   <property name="text">
    <string>Не выходить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="answer_1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answer_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answer_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answer_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answer_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="slider">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>22</width>
     <height>91</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="invertedAppearance">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="num_que">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>12</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="but_delete">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(199, 129, 129);</string>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_que">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>291</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(199, 129, 129);</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="var_ans_1">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>211</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="var_ans_3">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>271</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="var_ans_4">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>301</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="var_ans_2">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>241</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="var_ans_5">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>331</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="debug">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_que">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 160, 160, 150);</string>
   </property>
   <property name="text">
    <string>Закрытый</string>
   </property>
  </widget>
  <widget class="QPushButton" name="one_var">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 160, 160, 150);</string>
   </property>
   <property name="text">
    <string>Несколько</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_que">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>but_create_test</zorder>
  <zorder>but_sign</zorder>
  <zorder>but_reg</zorder>
  <zorder>but_info</zorder>
  <zorder>info_text</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>but_info_back</zorder>
  <zorder>name</zorder>
  <zorder>but_reg_enter</zorder>
  <zorder>but_reg_back</zorder>
  <zorder>line_reg_login</zorder>
  <zorder>line_reg_password</zorder>
  <zorder>line_reg_password_2</zorder>
  <zorder>text_reg_login</zorder>
  <zorder>text_reg_password</zorder>
  <zorder>text_reg_password_2</zorder>
  <zorder>but_sign_back</zorder>
  <zorder>but_sign_enter</zorder>
  <zorder>text_sign_password</zorder>
  <zorder>text_sign_login</zorder>
  <zorder>line_sign_password</zorder>
  <zorder>line_sign_login</zorder>
  <zorder>but_old_test</zorder>
  <zorder>but_new_test</zorder>
  <zorder>but_tmain_back</zorder>
  <zorder>line_name_test</zorder>
  <zorder>label_name_test</zorder>
  <zorder>info_text_2</zorder>
  <zorder>but_exit</zorder>
  <zorder>but_no_exit</zorder>
  <zorder>but_next</zorder>
  <zorder>but_pre</zorder>
  <zorder>answer_2</zorder>
  <zorder>answer_3</zorder>
  <zorder>answer_4</zorder>
  <zorder>answer_5</zorder>
  <zorder>slider</zorder>
  <zorder>num_que</zorder>
  <zorder>but_delete</zorder>
  <zorder>text_que</zorder>
  <zorder>var_ans_1</zorder>
  <zorder>var_ans_3</zorder>
  <zorder>var_ans_4</zorder>
  <zorder>var_ans_2</zorder>
  <zorder>var_ans_5</zorder>
  <zorder>debug</zorder>
  <zorder>close_que</zorder>
  <zorder>one_var</zorder>
  <zorder>answer_1</zorder>
  <zorder>line_que</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
